source/views/guides/group-by-and-queryable-by-name.html.slim

```diff
@@ -0,0 +1,118 @@
+section.banner
+  .content-wrapper
+    .banner-wrapper
+      h2.banner__heading Group By and QueryableByName
+
+main
+  section.demo
+    .content-wrapper
+      .guide-wrapper
+        markdown:
+          Diesel can't statically check all queries due to the limitless
+          combinations of SQL. When you can't represent your query with
+          Diesel's DSL, you can fallback to
+          [`sql_query`](http://docs.diesel.rs/diesel/fn.sql_query.html).
```

Comment:
Just a general remark: Using ref links and [semantic linefeeds](http://rhodesmill.org/brandon/2012/one-sentence-per-line/) make reviewing markdown (diffs) much easier. You don't have to change it here, though. :)